src/content/edit-user-script.js

```diff
@@ -173,3 +217,4 @@ window.addEventListener('beforeunload', event => {
     event.preventDefault();
   }
 });
+rivets.bind(document, gTplData);
```

Comment:
Should be blank lines, but why did this need to move all the way to the bottom?

Reply:
Not really sure why I stuck it all the way at the bottom. Should only need to be below "tplData"